rastervision/data/label_store/semantic_segmentation_raster_store.py

```diff
@@ -96,6 +110,20 @@ def save(self, labels):
 
         upload_or_copy(local_path, self.uri)
 
+        for vo in self.vector_output:
+            uri = vo['uri']
+            mode = vo['mode']
+            local_geojson_path = get_local_path(uri, self.tmp_dir)
+            if mode == 'buildings':
+                import mask_to_polygons.vectorification as m2p
+
+                with rasterio.open(local_path, 'r') as dataset:
+                    mask = dataset.read(1)
```

Comment:
This only will work with single band GeoTIFF output (as opposed to RGB output). Maybe we could avoid this issue by just doing the `m2p` earlier in this method when we have the raw predictions in RAM in single band format. Also, does this work with multiple classes? What assumptions does it make about the class_ids?

Reply:
If it's going to be a lot of work to make it more flexible you can make a separate issue for it, but just make sure to document the limitations and add input validation in this PR (ie. check that the vector output config is consistent with the `with_rgb` setting and the `class_map`

Reply:
Done.